api/envoy/config/filter/network/mysql_proxy/v1alpha1/mysql_proxy.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+package envoy.config.filter.network.mysql_proxy.v1alpha1;
+option go_package = "v2";
+
+import "validate/validate.proto";
+
+// [#protodoc-title: MySQL proxy
+// MySQL Proxy :ref:`configuration overview <config_network_filters_mysql_proxy>`.
+
+message MySQLProxy {
+  // The human readable prefix to use when emitting :ref:`statistics
+  // <config_network_filters_mysql_proxy_stats>`.
+  string stat_prefix = 1 [(validate.rules).string.min_bytes = 1];
+
+  // [#not-implemented-hide:] The optional path to use for writing MySQL access logs.
+  // If the access log field is empty, access logs will not be written.
```

Comment:
While this is not rendered now. Let's consider using warning/attention block for this.